pkg/functions/generate.jsonnet

```diff
@@ -85,17 +88,14 @@ local definitionStatus = k8s.GenerateSchema(
           additionalPrinterColumns: k8s.FilterPrinterColumns(version.additionalPrinterColumns),
         },
       ],
-      // defaultCompositionRef: {
-      //   name: k8s.GetDefaultComposition(s.config.compositions),
-      // },
     },
   },
 } + {
   ['composition-' + composition.name]: {
     apiVersion: 'apiextensions.crossplane.io/v1',
     kind: 'Composition',
     metadata: {
-      name: "composite" + composition.name + "." + s.config.group,
+      name: composition.name,
```

Comment:
think we dont need this change ?

Reply:
We do since the [defaultCompositionRef](https://github.com/crossplane-contrib/x-generation/pull/26/files#diff-94ab8ac1c4683ecd6c55a4659434e3da2d45e8287732cdeea201d4271bc3dd1fR53) targets the Composition from `generate.yaml`'s `compositions` `name` key whose `default` key is `true`.

Reply:
If you don't like the fact that all the example `package/compositions` `name` fields changed, I can update their respective `generate.yaml`s accordingly with the `"composite" + composition.name + "." + s.config.group` if you would prefer.

Reply:
Yes please Update - then other folks are aware of this change

Reply:
Pushed!